www/js/model.js

```diff
@@ -175,7 +218,31 @@ define([
 		};
 
 		obj.authored = lang.mixin({
+			addNode: function(options){
+				obj._updateNextXYPosition();
+				var newNode = lang.mixin({
+					ID: "id" + obj._ID++,
+					genus: "required",
+					accumulator: false,
```

Comment:
Professor, in our discussion had said it to be accumulator. This is what is in the data file.